gm4_live_catch/data/gm4_live_catch/functions/fish/tropical_fish/update.mcfunction

```diff
@@ -0,0 +1,10 @@
+# run from gm4_live_catch:fish/FISH_TYPE/summon
+
+
+# update
+data modify entity @s Air set value 60
+data modify entity @s Health set value 1f
```

Comment:
You have a storage set up, so writing these NBT values into storage would be more efficient, as the storage is merged onto the entity anyways.

Reply:
```suggestion
data merge storage gm4_live_catch:temp {target:{Air:60,Health:1f}}
```